serve/filters/filter/tx.go

```diff
@@ -0,0 +1,101 @@
+package filter
+
+import (
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+)
+
+type Options struct {
+	Height    int64
+	Index     uint32
+	GasUsed   struct{ Min, Max int64 }
```

Comment:
What do you think about defining a type `GasLimits` that has fields `Min` and `Max`?

Reply:
These should be pointer values, so if they're not set, we can assume they are not checked against

Reply:
It certainly seems effective. I'll modify it to use pointers

Reply:
added the `GasLimit` filter which use the value stored in `GasUsed`